File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/PhotosModel/ResponsePhotoModelClass.java

package restaurant.sa.com.sarestaurant.appview.restaurant.model.PhotosModel;

import java.util.List;
import com.google.gson.annotations.SerializedName;
public class ResponsePhotoModelClass {

    @SerializedName("html_attributions")
    private List<Object> mHtmlAttributions;
    @SerializedName("result")
    private Result mResult;
    @SerializedName("status")
    private String mStatus;

    public List<Object> getHtmlAttributions() {
        return mHtmlAttributions;
    }

    public void setHtmlAttributions(List<Object> htmlAttributions) {
        mHtmlAttributions = htmlAttributions;
    }

    public Result getResult() {
        return mResult;
    }

    public void setResult(Result result) {
        mResult = result;
    }

    public String getStatus() {
        return mStatus;
    }

    public void setStatus(String status) {
        mStatus = status;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/PhotosModel/ResponsePhotoModelClass.kt
package restaurant.sa.com.sarestaurant.appview.restaurant.model.PhotosModel

import com.google.gson.annotations.SerializedName

class ResponsePhotoModelClass {

    @SerializedName("html_attributions")
    var htmlAttributions: List<Any>? = null
    @SerializedName("result")
    var result: Result? = null
    @SerializedName("status")
    var status: String? = null

}
